### DISCUSSION TITLE
wishlist: Add multiuser support

### DISCUSSION THREAD

**Discussion by ftoledo** [vit1251/golden#121] [Ideas]
Just an indea:

Add auth layer to manager multipler users with ftn addres as login user 

Set global config root path like /home/GoldenUsers

Handle the session for every user that are logged.

Each user that logeed in, have they config path for load each settings ex:
```
ragnarok@ragnarok:~$ tree GoldenUsers/
GoldenUsers/
├── 4_902_123
├── 4_902_26_5
└── 4_902_999_999
```
Restrict some options to avoid users changes.. eje (node/point address, Services page, boos address)

**Comment by vit1251** [comment]
As a FTN point user, I may use several network FTN addresses. But in this case we are faced with the increasing complexity of routing EchoMail and Netmail messages and processing TIC's requests. As a result, we will combine echoes. It's about the same as phones with two SIM cards - it's almost never clear what will happen and from which card.

I think that at the moment, you can use the ability to change the user through `su` or `sudo` application to switch between OS user.

Next level is point application is `Golden Node` application with routing, multiuser user processing and provide web reading and write gateway, but it not in this repository and our current discussions.

**Reply by ftoledo** [reply]
Hi thanks for your reply!
I was also thinking of something simpler, put multiple instances listening on different ports (simply protect them with http auth). It could be using docker. But I do not know if it is an optimal solution to be able to scale.

Unfortunately I have no skill in go, although I am reviewing the code a bit and learning.

**Reply by vit1251** [reply]
You may use command line arguments to change home directory and port, but it violate with main goal Point project be simple solution. 

GP designed for easy use and setup. Using command line and configuring multiply path is require some advanced skills with shell and not easy for newbe users.
